### PR TITLE
Cleanup package.json

### PR DESCRIPTION
**Pull request by SimenB**
Ref #291

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```
Comment (SimenB) [inline .gitattributes]: I'm on a Windows machine at the moment, adding this keeps CRLF out of the repo...

File: package.json [hunk 2]
```diff
@@ -1,36 +1,56 @@
...
-  "directories": {
-    "lib": "bin"
+  "description": "A jQuery plugin that offers a simplistic way of implementing Lion OS scrollbars.",
+  "version": "0.8.5",
+  "author": "James Florentino <jamesflorentino@gmail.com> (http://blog.jamesflorentino.com/)",
+  "bugs": {
+    "url": "https://github.com/jamesflorentino/nanoScrollerJS/issues"
   },
+  "contributors": [
+    "Krister Kari (http://krister.fi/)"
```
Comment (SimenB) [inline package.json]: Didn't see an email on your profile here at GitHub

**Comment by kristerkari** [comment]
Thanks! I'll test this out and if everything is ok ship with next version 0.8.6

**Comment by SimenB** [comment]
@kristerkari Great :smile: 
This is mostly just adding all the missing fields, then running it through https://github.com/HenrikJoreteg/fixpack

**Pull request merged**